source/advanced_components/docker/docker_integration.rst

```diff
@@ -0,0 +1,158 @@
+===========================================
+Docker and OpenNebula integration
+===========================================
+
+For docker integration we need to install docker machine and docker at OpenNebula frontend:
```

Comment:
We should say here that the following instructions are ubuntu (which version) specific